### PR TITLE
Move 2j3 changes to 2k6

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
As spotted by @Laura-O at #922 , the A1c movement makes more sense under 2k.

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -90,14 +90,14 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
-    - 2j3) If a competitor does not fulfill the event's requirements (e.g. know how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result.
 - 2k) At the discretion of the WCA Delegate, a competitor may be disqualified from some events (a single event, multiple events, or all events) if the competitor:
     - 2k1) Fails to check in or register in time for the competition.
     - 2k2) Is suspected of cheating or defrauding the officials during the competition.
         - 2k2a) The WCA Delegate may disqualify any suspected results.
     - 2k3) Behaves in a way that is unlawful/violent/indecent, or intentionally damages venue facilities or personal property within the venue.
     - 2k4) Distracts or interferes with others during the competition.
     - 2k5) Fails to abide by WCA Regulations during the competition.
+    - 2k6) Does not fulfill the event's requirements (e.g. not knowing how to solve the puzzle). A competitor must not compete with the expectation of a DNF result or an intentionally poor result.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md:100]: Does the example make sense of being negative?

**Comment by xsrvmy** [comment]
I have a suggestion for splitting 2k6 into 2k6 and 2k7
2k6) Does not fulfill the event's requirements
2k7) competes with the intention of wasting competition resources (eg. competing with the expectation of an intentionally poor result, intentionally makes an FMC result hard to grade)

**Comment by xsrvmy** [comment]
Also there is a problem with placing the change here: this would allow the delegate to disqualify the competitor from the entire competition (but A1c does not)

**Comment by Ivan-Brigidano** [comment]
> I have a suggestion for splitting 2k6 into 2k6 and 2k7
> 2k6) Does not fulfill the event's requirements
> 2k7) competes with the intention of wasting competition resources (eg. competing with the expectation of an intentionally poor result, intentionally makes an FMC result hard to grade)

@xsrvmy I think it makes sense to have both in a single regulation as both phrases are related.

> Also there is a problem with placing the change here: this would allow the delegate to disqualify the competitor from the entire competition (but A1c does not)

I don't think this is a big deal as it's still Delegate discretion. I don't think this might cause any problems as any Delegate would disqualify a competitor from the entire competition because of not knowing how to solve a specific event.

**Comment by xsrvmy** [comment]
We still need to mention the FMC issue.

**Comment by Ivan-Brigidano** [comment]
> We still need to mention the FMC issue.

What do you mean here?

**Comment by lgarron** [comment]
> @xsrvmy I think it makes sense to have both in a single regulation as both phrases are related.

Agreed.

> I don't think this is a big deal as it's still Delegate discretion. I don't think this might cause any problems as any Delegate would disqualify a competitor from the entire competition because of not knowing how to solve a specific event.

I also agree with this. It's very rare for a Delegate to disqualify a competitor from the entire competition, and I think it's unlikely to happen by "accident". However, it may actually be appropriate in this situation, e.g. if a competitor has signed up for 3x3x3 followed by 3x3x3 BLD but clearly can't even solve a cube sighted.

> > We still need to mention the FMC issue.
> 
> What do you mean here?

I think @xsrvmy might mean adding this to 2k7?  I don't think it's really needed.

File: wca-regulations.md (hunk 1 above)
Comment (lgarron) [reply wca-regulations.md:100]: I think it's good to stay similar to the old A1c, since it still works well enough.

**Review by lgarron — APPROVED** [review]

**Comment by xsrvmy** [comment]
Just noticed a different issue: A1c allows an individual attempt to be disqualified. 2k only allows disqualification from the event. Which one takes precedence?

**Comment by Ivan-Brigidano** [comment]
> Just noticed a different issue: A1c allows an individual attempt to be disqualified. 2k only allows disqualification from the event. Which one takes precedence?

I don't think there is a problem with that. I think both regulations are compatible

File: wca-regulations.md (hunk 1 above)
Comment (Ivan-Brigidano) [reply wca-regulations.md:100]: Notice that 2k force to make it negative. I think the change makes sense

**Pull request merged**